### ISSUE TITLE
Update never finishes downloading

### ISSUE THREAD

**Issue by JordyGit** [microsoft/BotFramework-Emulator#2347]
## Version
V4.14.0
![image](https://user-images.githubusercontent.com/33500507/144566840-7ba51b1c-f536-4f02-8423-4c0d3c18554e.png)

## Describe the bug
On startup the emulator asks me to update and gives me 3 options where I selected the option to download/install the new version now. The window with "Downloading..." and a Dismiss button appears. There is no more feedback about the download (no download speed, no progress feedback, nothing) and it keeps this way. I left this for a whole day untouched and it still didn't complete. The only thing I can do is dismissing it with that button.

## To Reproduce
1. Have an older emulator version (V4.14.0)
2. On Startup it will ask to update its version, select to do that now
3. Downloading window appears but never finishes (bug)

## Expected behavior
Download window should give feedback about the download, for example the progress as a percentage. 
If something is/goes wrong it should give feedback instead of doing nothing.

## Screenshots
![image](https://user-images.githubusercontent.com/33500507/144567641-d44e7fd5-cddf-4be8-b347-d8b650358a71.png)

**Comment by dmvtech** [comment]
Not sure if related, but when I select `Check for update...`, I get no response from UI at all. Currently running `4.14.0`.

**Comment by anishprasad01** [comment]
The same issue occurred for me as well when attempting to upgrade from `4.14.0`. I ended up uninstalling it, downloading the installer for the latest version, and reinstalling to get to `4.14.1`.

**Comment by axelsrz** [comment]
Hello @tonyanziano could you take a look at this?

**Comment by benbrown** [comment]
@JordyGit We are taking a look into this today.

**Comment by tonyanziano** [comment]
I've repro'd this on my end, and I found the following error message which seems to be the culprit:

`An error occurred in the updater's "update-available" event handler: Error: There was an error while trying to download the latest update: Error: ENOENT: no such file or directory, open 'C:\Users\tonya\AppData\Local\Programs\Bot Framework Emulator\resources\app-update.yml'`

I'm unclear on why this is an issue now when it never was before. I would assume that it has something to do with the Electron upgrade, but this issue is present in 4.14.0 which is still on the older version of Electron.

**Comment by tonyanziano** [comment]
As a workaround for now, you can go to your Emulator installation directory, and create a file called `app-update.yml` at the path:

`<Emulator-install-dir>/resources/app-update.yml`

with the content:

```yaml
updaterCacheDirName: '@bfemulatormain-updater'
```

I just tested an update after adding this file, and it seems to have unblocked the update and everything worked as expected.

**Comment by tonyanziano** [comment]
Note to fixer: We need to have this file generated as part of our build pipeline. Either by manually copying it in or by getting `electron-builder` to generate it again.

**Comment by ivorb** [comment]
@JordyGit understood, that works or reinstalling with newer version works, but not a good experience for our learners doing labs. Is the idea that future versions won't hit this, and this issue for 4.14.0 is a won't fix?
